packages/ng/libraries/root/src/style/components/_popup.scss

```diff
@@ -102,3 +101,22 @@
 		opacity: 0;
 	}
 }
+
+@include media_smaller_than("sm") {
+	.lu-popup-panel {
+		border-radius: 0;
+		max-width: 100vw;
+		max-height: 100vh;
+		width: 100vw;
+		width: fill-available;
```

Comment:
Why not a 100% width and height?

Reply:
Rather than 100vw and 100vh ? It should allow us to go fullscreen whatever the parents' style is. But might be overkill

Reply:
Height 100% is dangerous if the parent doesn't have a height set. I'd rather keep it to 100vh and therefore, to have a logic between the two sizes, keep width to 100vw. What do you think ?

Reply:
I'm afraid this will cause conflicts with browsers that don't always include scrollbar calculations.

Reply:
It all depends if we can make sure that the popins are always before the </body> taf in the DOM. Normally yes.

Reply:
I will switch width to 100%, but might keep height to 100vh and fill-available (for chrome mobile...)